index.bs

```diff
@@ -851,6 +857,49 @@ spec:css-syntax-3;
         origin. To set that flag, developers should call {{preventSilentAccess()}}.
   </div>
 
+  ### UI Mode Requirements ### {#ui-mode-requirements}
+
+  When making a request via {{get(options)}}, developers can set a case-by-case requirement for the
+  desired type of [=user mediation=] by choosing the appropriate {{CredentialUiModeRequirement}} enum value.
+
+  Note: This allows callers to distinguish between different available forms of [=user mediation=], when it occurs. Whether
+  or not it occurs depends on the value of {{CredentialMediationRequirement}} for the request. Not all values of
+  {{CredentialUiModeRequirement}} make sense in combination with all values of {{CredentialMediationRequirement}}, and the
+  interaction depends on the `[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)` methods for credential
+  types that support it.
+
+  <pre class="idl">
+    enum CredentialUiModeRequirement {
+      "active",
+      "immediate"
+    };
+  </pre>
+  <div dfn-for="CredentialUiModeRequirement" dfn-type="enum-value">
+    :   <dfn>active</dfn>
+    ::  If [=user mediation=] occurs for the given operation, then the user agent will show a modal dialog to the user
+        unless the selected {{CredentialMediationRequirement}} specifies otherwise.
+
+        Note: This is the default {{CredentialUiModeRequirement}} value. It is intended to preserve existing user agent 
+        behavior from before this enumeration was defined, which is that all {{CredentialMediationRequirement}} values
+        cause modal dialogs to be displayed except for {{CredentialMediationRequirement/conditional}}.
```

Comment:
`IdentityCredential`'s passive mode also does not result in a modal dialog.

The default should probably be unspecified, so we can support the fact that FedCM does passive by default and every other credential provider active.

(Discussed offline as well.)